assets/services.json

```diff
@@ -13,81 +13,297 @@
+        "||amazonaws-china.com^",
+        "||amazonaws.cn^",
+        "||amazonaws.co.uk^",
+        "||amazonaws.com.cn^",
+        "||amazonaws.com^",
+        "||amazonaws.tv^",
+        "||amazonbusiness.cn^",
+        "||amazonbusiness.com.cn^",
+        "||amazonbusiness.org^",
+        "||amazonbusinessblog.com^",
+        "||amazonchoice.cn^",
+        "||amazonchoice.com.cn^",
+        "||amazonchoices.cn^",
+        "||amazonchoices.com.cn^",
+        "||amazondevicesupport.com^",
+        "||amazonfctours.com^",
+        "||amazonianblog.com^",
+        "||amazonimages.com^",
+        "||amazoninspire.cn^",
+        "||amazoninspire.com.cn^",
+        "||amazonlaunchpad.cn^",
+        "||amazonlaunchpad.com.cn^",
+        "||amazonlaunchpad.com^",
+        "||amazonlending.com.cn^",
+        "||amazonliterarypartnership.com^",
+        "||amazonlumberyard.wang^",
+        "||amazonnow.cn^",
+        "||amazonnow.com.cn^",
+        "||amazonpay.com^",
+        "||amazonpay.in^",
+        "||amazonprimevideo.cn^",
+        "||amazonprimevideo.com.cn^",
+        "||amazonprimevideos.com^",
+        "||amazonsdi.com^",
+        "||amazonses.com^",
+        "||amazonstudiosguilds.com^",
+        "||amazontrust.com^",
+        "||amazonvideo.cc^",
+        "||amazonvideo.com^",
+        "||amazonvideodirect.com^",
+        "||amazonwebservices.com.cn^",
+        "||amazonworkdocs.cn^",
+        "||amazonworkdocs.com.cn^",
+        "||amazonworkdocs.com^",
+        "||amplifyapp.com^",
+        "||amplifyframework.com^",
+        "||amzn.asia^",
+        "||amzn.com^",
+        "||amzn.to^",
+        "||amznl.com^",
+        "||asfiovnxocqpcry.com.cn^",
+        "||assoc-amazon.cn^",
+        "||associates-amazon.com^",
+        "||audible.com^",
+        "||aws-border.cn^",
+        "||aws-icp-domain-manager.cn^",
+        "||aws-iot-hackathon.com^",
+        "||aws^",
+        "||awsapps.cn^",
+        "||awsapps.com.cn^",
+        "||awsautopilot.com^",
+        "||awsautoscaling.com^",
+        "||awsbraket.com^",
+        "||awscommandlineinterface.com^",
+        "||awsedstart.com^",
+        "||awseducate.com^",
+        "||awseducate.net^",
+        "||awseducate.org^",
+        "||awsglobalaccelerator.com^",
+        "||awsloft-johannesburg.com^",
+        "||awsloft-stockholm.com^",
+        "||awssecworkshops.com^",
+        "||awsstatic.cn^",
+        "||awsstatic.com^",
+        "||awsthinkbox.com^",
+        "||awstrack.me^",
+        "||awstrust.com^",
+        "||boxofficemojo.com^",
+        "||cdkworkshop.com^",
+        "||cloudfront-cn.net^",
+        "||cloudfront-test.cn^",
+        "||cloudfront.cn^",
+        "||cloudfront.net^",
+        "||containersonaws.com^",
         "||createspace.com^",
-        "||aws"
+        "||elasticbeanstalk.com^",
+        "||gameon-masters.com^",
+        "||gdansk-amazon.com^",
+        "||images-amazon.com^",
+        "||imdb.com^",
+        "||imdb.to^",
+        "||imdb^",
```

Comment:
`imdb` TLD?

Reply:
I removed all domain names which are belonging to `imdb`.

Reply:
https://icannwiki.org/.imdb